.nuxt/utils.js

```diff
@@ -393,7 +393,7 @@ function tokensToFunction(tokens) {
         continue
       }
 
-      var value = data[token.name]
+      var value = data[token.name || 'pathMatch']
```

Comment:
**question**: was this change intended?

Reply:
not intentional - happened with  `npm run dev`. fixed with `git checkout master -- .nuxt/utils.js `